Strings/AppClip.jsonc

```diff
@@ -2,66 +2,66 @@
 
 	"スキャン中はカードと iPhone を動かさないようにします。カードによってスキャンが完了するまで時間がかかる場合があります。": {
 		"ja": "スキャン中はカードと iPhone を動かさないようにします。カードによってスキャンが完了するまで時間がかかる場合があります。",
-		"en": null,
-		"zh-Hans": null,
-		"zh-Hant": null,
-		"zh-HK": null
+		"en": "Do not move the card or your iPhone during the scan. Depending on the card type, this process could take some time to complete.",
+		"zh-Hans": "扫瞄期间请勿移动卡片或您的 iPhone。这个过程可能需要一段时间才能完成。",
+		"zh-Hant": "掃瞄期間請勿移動卡片或您的 iPhone。此程序可能要一段時間才能完成。",
+		"zh-HK": "掃瞄期間切勿移動卡片或您的 iPhone。過程可能需要一段時間才能完成。"
 	},
 
 	"フルバージョンの App ではカードに記録されている利用履歴も確認できます。": {
 		"ja": "フルバージョンの App ではカードに記録されている利用履歴も確認できます。",
-		"en": null,
-		"zh-Hans": null,
-		"zh-Hant": null,
-		"zh-HK": null
+		"en": "You can check the transaction history on the card in the full app.",
+		"zh-Hans": "您可于完整的 App 内查询卡片上的交易记录。",
+		"zh-Hant": "您可於完整的 App 內查詢卡片上的交易記錄。",
+		"zh-HK": "您可於完整的 App 內查詢卡片上的交易記錄。"
 	},
 
 	"金属製の机の上やケース・バンパーを装着している場合、電波干渉のために正しく読み取れない場合があります。\\nイヤホンや充電等のためにケーブルを接続している場合、読み取りが完了しない可能性があります。": {
 		"ja": "金属製の机の上やケース・バンパーを装着している場合、電波干渉のために正しく読み取れない場合があります。\\nイヤホンや充電等のためにケーブルを接続している場合、読み取りが完了しない可能性があります。",
-		"en": null,
-		"zh-Hans": null,
-		"zh-Hant": null,
-		"zh-HK": null
+		"en": "If you have a metal desk or you use a metal case on your iPhone, scanning may be affected due to wireless interference.\\nIf your iPhone is connected to a cable, scanning may not be able to complete.",
```

Comment:
> If your iPhone is connected to a cable, scanning may not be able to complete.
isn't very natural, I'd like to recommend;
> Scanning may also be interrupted if your iPhone is connected to a cable.

But I'd like some more info on this from @treastrain as to why it cannot be completed while a cable is connected. Electrical interference? Maybe we should also say that you cannot scan while Wireless Charging... (to cover all bases)

In that case, maybe this would work better;
> Additionally, scanning may be interrupted due to electrical interference if the device is being charged.

Reply:
I placed my iPhone next to my WiFi router and the IC card was read successfully, and even tried scanning with the charger plugged in and my phone sitting on my power board... but they were all read successfully. I think the original complaint is due to concerns of wireless interference. I still think this is a niche problem, but we can catch all of that in one translation;

> Card scanning may be unexpectedly interrupted due to factors such as wireless interference. Avoid scanning on or near metal objects, devices that emit strong radio waves (like WiFi routers), or induce magnetic currents (like phone chargers).

Let me know what you think. Of course the other translations would need to be modified to reflect this new definition as well.

Reply:
Thanks. As a result, I think the sentence @kurisubrooks you suggest is the smartest and best. However, I suggest that you modify "metal objects" a little and add expressions such as "cases, bumpers, desks, etc."

`Card scanning may be unexpectedly interrupted due to factors such as wireless interference. Avoid scanning on or near metal objects (case, bumper, or desk), devices that emit strong radio waves, or induce magnetic currents.`

(I forgot about the MagSafe charger... 😢)

Reply:
Because It says "avoid scanning **on** metal objects", I think people should understand that it includes metal desks, but I added the other two.

> Card scanning may be unexpectedly interrupted due to factors such as wireless interference. Avoid scanning on or near metal objects (including with metal bumpers or cases), devices that emit strong radio waves, or induce magnetic currents.

I think the definition has become slightly complicated, so it troubles me. There might be a better way to word it, so I'll ask @w41k3rc0m for a review once again. 

Also, should we also include examples for radio and magnetic waves? Like WiFi router, microwave oven, etc...? Though it might only make it more complex...

By the way, I think MagSafe could be a problem, because of the NFC coil. Do you have the iPhone 12? We should test it. @treastrain

Reply:
I do not believe it is necessary to provide detailed examples of electromagnetic waves. And I don't plan on getting an iPhone 12 series right now, so I hope that Apple is thinking well about MagSafe and NFC...

Reply:
Thanks. I'll change it to @kurisubrooks' version.

_(Sorry for the late response, was busy with other stuff)_